### PR TITLE
apify-cli 0.19.0

### PR DESCRIPTION
**Pull request by apify-service-account**
Automatic update of the `apify-cli` formula.

  CC @fnesveda @B4nan

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  ==> /home/linuxbrew/.linuxbrew/Cellar/apify-cli/0.19.0/bin/apify init testing-actor
  Warning: The current directory does not look like a Node.js or Python project.
  ? Do you want to continue? (Y/n) 
  Error: apify-cli: failed
```

**Comment by chenrui333** [comment]
CC @fnesveda @B4nan

File: Formula/a/apify-cli.rb [hunk 1]
```diff
@@ -36,10 +36,10 @@ def install
   end
 
   test do
-    # Test that the Apify CLI is at all installed and working
-    assert_match "apify-cli/#{version}", shell_output("#{bin}/apify --version")
-    # Test that the CLI can initialize a new actor
-    system "#{bin}/apify", "init", "testing-actor"
-    assert_predicate testpath/".actor/actor.json", :exist?
+    output = pipe_output("#{bin}/apify init testing-actor", "Y")
```
Comment (B4nan) [inline Formula/a/apify-cli.rb:39]: we got some issues in the automated pipeline, but this should fix the test

```suggestion
    output = pipe_output("#{bin}/apify init -y testing-actor", "Y")
```

(we no longer allow running this command in empty folder without confirmation)

Comment (chenrui333) [reply Formula/a/apify-cli.rb]: works for me 👍

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**